### DISCUSSION TITLE
Thinking Cap #3 💡

### DISCUSSION THREAD

**Discussion by albertkun** [albertkun/24SU-ASIAAM-191A#40] [Thinking Caps]
### Respond to the prompt below in at least 2-3 sentences
> ## Consider what are your ethical views when it comes to society and technology? How do your ethics empower civil society? 

Completing the [readings](https://albertkun.github.io/24SU-ASIAAM-191A/assignments/week3/reading/) will be particularly helpful in this thinking cap.

#### Note: Responses submitted after Sunday 3pm will not be eligible for awards.

Post your thinking cap response and reply to others in this thread. 

##### Refer to this [documentation](https://albertkun.github.io/24SU-ASIAAM-191A/assignments/week1/thinking_cap) if you need a reminder on how to complete the assignment.

**Comment by stinest** [comment]
We push the boundaries of technology and advance in new ways every year. However, it is because of its almost limitless capabilities, that I believe it is incredibly important that we _don't_ explore its borders. Technology should be used to benefit humanity, rather than exploit it for all it has to offer simply because it can. To elaborate, technology is often used unethically. In _The Ethics of GIS_ by Jeremy Crampton, he discusses the societal implications behind surveillance in relation to GIS usage. Although spatial data has become increasingly common, "many are unwilling to accept that maps could have an external, ideological agenda in addition to their role as representations of the environment" (pg. 85). Unethical practices such as surveillance don't neutrally represent its environment, but instead infringes upon our privacy and can harm individuals. I believe technology needs to be used with a sense of responsibility, prioritizing inclusivity and the well-being of society over raw technological advancement. True progress in technology cannot exist without corresponding advancements in humanity.

**Reply by sjeyakum** [reply]
I agree with you that technology, including surveillance, is not free of prejudices or unethical practices. The biased nature of these applications can be especially harmful if they are used to target certain groups for corrupt or capitalistic purposes. However, I also feel that for these reasons, such matters are difficult to discuss when addressing ethics because it is not appropriate to simply label technology or broad practices, like surveillance, as completely ethical or unethical and make decisions based on that. For instance, surveillance is often a fundamental activity of public health initiatives, so saying it is totally unethical, harmful, and should be discontinued feels too extreme. At the same time, disregarding the ethics of a practice that appears to be driven solely by data or technology (e.g., surveillance surveys) neglects unethical behavior like discriminatory targeting. I guess my point is that when discussing the ethics of technology in society, you must consider its design, platform, and applications. I feel like this connects with your concluding statement that progress cannot be made unless both sides grow and work together. Creators and community members must collaborate and discuss ethics together to ensure that ethical views are not purely based on one side's position, power, or preferences.

**Reply by kimada62** [reply]
It is so sad to think that a map, which most think of as a neutral source of information, can be used for ulterior motives. Conversations around ethics must continue to grow, because the technology is constantly changing and its unethical surveillance usages should not go unchecked.

**Comment by pwinnie2026** [comment]
Technology will continue to develop, but our conversations on ethics and our handling of deep-rooted social issues are not developing fast enough in my opinion. We touched on this briefly in the lecture and in Paris Marx's article "Don't Blame Social Media. Blame Capitalism", those heavily aligned with Technological Determinism believe in technological development for the betterment of humankind. While in some respects, this can be true, artificial intelligence powering cancer research models can also power and exacerbate genocides. In my personal experience within the "tech space", people are eager to learn new AI tools as "the future" whilst not always prioritizing or discussing ethics. In an AI/ML program currently, most of the foundational course is simply on learning how to code through different Python packages. Especially with the U.S. Supreme Court's ruling on Chevron Deference, I am a little wary of AI regulation on a nationwide level. Not only is technical knowledge important but in such lawmaking positions, our lack of conversations highlighting ethics and technology is something that should change. My involvement with SWE@UCLA's Lobbying Committee is one space where we try to highlight this on a committee level. The only issue is, that it is harder to highlight such topics when we open our meetings up to wider SWE given SWE's neutrality and corporate/institution-friendly profile. We met with SWE's national lobby director during our trip to DC; her conversations about working in the Defense, while interesting to hear, felt sometimes removed from the human perspective.

**Reply by kianaptran** [reply]
Yes! It pains me to hear how money-driven research has become. Although it would be ideal for research to function without the pressures of funding and profiting, this is not the case. I cannot even imagine the amount of information that can help solve global issues like cancer that are gatekept to ensure their research company remains profitable.

**Reply by Keiya-Wada** [reply]
I completely agree with what you said. When I am scrolling through instagram and see memes about curing cancer, a lot of the comments under those posts state that "the government will come get you if you find the cure to cancer" or something close to that. Although a bit exaggerated, this goes to show how truly financially driven companies are currently, rather than thinking about the well being of society.

**Comment by zhamic7** [comment]
When it comes to society and technology, I believe simply that the well-being of people should be prioritized. Social media is an interesting example of ethics and technology, because while I believe it's original intention of connecting people from all over the world was ethical (e.g. "old-fashioned" discussion forums), it's most popular form today has made it a weapon wielded by corporations and capitalists against civil society. In response to Paris Marx's article, even though I do agree that social polarization has largely been driven by capitalism and the profitability of political division, I'm not sure if I agree with the sentiment that we should therefore reduce our criticism of social media companies like Facebook and YouTube because capitalism is the ultimate issue and that social platforms utilizing personalization algorithms are not a major problem since extremists "were radicalized on smaller platforms without the same algorithmic curation or even profit motives like 4chan, 8chan, Gab, and Stormfront." While extremists certainly receive the most media attention, they are most certainly not the major drivers of today's political divide. How many of us today, both right-wing and left-wing, actually bother to look beyond what our social media feeds show us to learn about the circumstances influencing the actions and decisions of people we oppose politically instead of just reinforcing our previous biases that the other side consists of only crazy extremists who are incapable of logical reasoning and just generally bad, selfish people? I don't think social media in its current and most popular form could ever become an ethical tool even if the world suddenly shifted to a different -ism overnight because, ultimately, it has become a tool against people rather than for people; the bad actors right now may be capitalism and technocratic agendas, but unregulated technology like social media and big data analysis inherently have made it and will continue to make it much easier for the elite to abuse their power and influence over the rest of society. "Returning to a more non-commercial structure where public companies own key infrastructural pieces, cooperatives operate a range of platforms with far different incentives given the lack of profit motive, and average people can collaborate on new digital tools without a commercial imperative" doesn't seem much more ethical to me -- generative AI, for example, has been for a long time a non-commercial product of university-funded research and is supported by many "average" people, but hasn't it also always inherently been a degradation of people, especially artists and writers, and our creative work?

**Reply by kianaptran** [reply]
I completely agree that the current state of social media is only doing more harm than good. Although some aspects of social media, society, and technology can be used for good (e.g., providing platforms for organizations, connecting people, and increasing community presence), they would be subjected to heavy criticism, hate speech, exploitation, etc., which questions the use of technology in the first place. Many people have found that social media has hurt their mental health so much that they had to altogether remove themselves from it to gain clarity. This then makes me question if it is possible to minimize technology's harm without having to heavily regulate socials and technology or completely shutting it down.

**Reply by Keiya-Wada** [reply]
Yes, I completely agree with what you said. Although the initial idea of connecting people and allowing people to share their lives seems like a positive one, this has ultimately led to an extreme form of comparison, discouraging many with depression, body-dysmorphia, etc. Moreover, companies collect data from users, which is often interpreted as a negative thing. Because of these reasons, many parents actually ban the use of social media. If the impact of a certain technological tool becomes so bad that people have to ban the use of it, then I truly wonder if these social media platforms are needed in society.

**Comment by kimada62** [comment]
My ethical views with society and technology has only recently come to some level of maturity after experiencing and ethical dilemma myself. Last year, the SAG-AFTRA guild went on strike due to various issues. However, its most prominent concern had to do with establishing new AI policies. Crampton's reading which stated"'nailing down' an ethical code or standards of practice is not the solution" and that "solutions to ethical problems are sometimes as bad as the problems themselves," brought SAG-AFTRA's issue to mind for me. 

While Crampton in some ways mirror the SAG-AFTRA strike, because they are attempting to nail down a solution, I do think that finding a code of conduct is necessary. However, we are currently seeing the issues of his warnings of solutions sometimes being as bad as the problems.  

One of the main concerns was that performers were asked permission for digital replication before they were hired. While this sounds good, the studios would have permission to use their digital replicas on future projects without further requirements to pay the performer, because they stated their permission on a one time contract that could pay as little as a day's work. If a performer did not give permission for digital replication, the studio could just go to the next best performer who would agree to their terms. 

After the strike, some things with AI were "solved," but it still permitted loopholes for studios to take advantage of performers. For example, A list performers who make for $80,000 a year are permitted some leverage to negotiate digital replication. However, those who make less are at risk of a normalized deal with the studios that will inhibit their ability to control further uses of their digital replications.

So, in the code of conduct prior to the strike and even after the deal, we are seeing the consequences of solutions being as bad as the problems themselves. These large corporations will not find a solution without a code of conduct to protect performers, because they will always look for loopholes. So, I believe that Crampton is right in that the solutions are sometimes as bad as the problems. However, the union needs a nailed down solution, otherwise studios will find some way around it. 

I have shared my findings in a journalism class, so I hope that my ethics with this issue has brought awareness to it with new audiences. I have also attended the strike's picket lines and have had to fight for my contract on one  job to ensure I was being paid correctly (though that was an issue somewhat different from this specific topic).

**Reply by sjeyakum** [reply]
Thank you for sharing your thoughts and experiences with this issue! I appreciate how clearly and concisely you connected the Campton reading with your experience and insights regarding the SAG-AFTRA strike. It is indeed unfortunate to see disparities in treatment and hiring practices concerning digital replication and the use of AI. You make a good point that solutions can be as problematic as the issues they aim to solve, particularly when they are implemented hastily without opportunities for discussion, collaboration, or creativity in finding alternative methods. I think this issue also speaks to the problem-solving mindset that we have been breaking down in class over the past few weeks. Approaching technology and society with the mindset that one side's problems should be fixed or addressed by the other side (i.e., unfairly taking advantage of performers and their consent upon hiring to "solve" AI likeness issues) can be detrimental and unethical, especially without a robust code of conduct (as you mentioned) or proper regulations.

**Comment by sjeyakum** [comment]
Growing up, I often heard that technology is merely a tool, and its impact depends on how it is used. This idea was usually accompanied with the common knife analogy, such that a knife is just a tool as a surgeon can use it to save lives while a murderer can use it to harm others. However, I feel like using this same analogy to label technology as “just a tool” oversimplifies issues and presents a rather limiting view, especially in discussions about the ethics of technology's applications and performance in civil society. As such, I appreciated Jeremy Crampton’s comment in the article, "The Ethics of GIS" that questions whether GIS is just a tool or a way of thinking. I feel like this point highlights how this simple distinction significantly influences ethical debates, as labeling technology as a tool overlooks its transformative and influential potential in various contexts, while identifying technology as a social model of communication addresses its connections with systems of power, discriminatory practices, capitalism, corruption, progress, and social empowerment. I guess my point is that as more and more people engage with technology and it intertwines with societal interests and goals, it inevitably evolves beyond a mere tool. In that case, viewing technology as a model rather than a tool aligns with my belief that ethical considerations must address both societal and technological aspects, as neither side is completely guilty or innocent when it comes to social challenges. For instance, while I agree that issues like targeted surveillance include technological problems (i.e., programming errors, data-driven features, etc.), they are also the consequences of biased frameworks (i.e., institutional racism) and supported by diverse actors (i.e., policymakers, private corporations). As such, solutions that solely focus on correcting algorithms or other application features are too narrow-minded and disregard underlying social causes. Likewise, advocating for public changes in attitudes and applications without discussing the technology fails to truly empower community members, as technology and society are positioned in a complex, interdependent cycle where expecting social changes without addressing technological influences on power and control is unrealistic.

**Reply by stinest** [reply]
I think this is an extremely well-written and amazing reply to the prompt! Technology has become incredibly transformative and holds so many roles in modern society, evolving beyond just being a tool. I most agree with your last point in achieving societal changes without considering how technology influences power is unrealistic.

**Reply by zhamic7** [reply]
I also agree with your point that labeling technology as "just a tool" is an oversimplification. Ideally, I think this is what technology should have been, but unfortunately, the obsession with technology by governments/corporations as well as in media has made it a significant part of today's dominant discourse. Even if the core problems lie within our social structure, technology has also become another major driver of these issues.

**Comment by kianaptran** [comment]
Before reading this week's articles, I had an extremely simplistic perspective on ethics regarding society and technology; similar to the shared themes found in Crampton's article, I believed that technology is merely a collection of data used to represent society and as long as the internal ethics of the data was preserved (i.e., data was ethically sourced, used for "good," kept private, etc.) then its practice would be considered ethical. However, after reading Crampton's article on the ethics of GIS, I became more aware of the external ethical implications of cartography. More specifically, I was surprised at how easily aggregated data could be traced. In the reading, it only took about "£2.50 per 1,000 names from the local government authority" (Crampton 88). Additionally, Marx's article on Capitalism's influences on social media further emphasizes the lack of ethics in modern technology. Navigating technology without Capitalism's monetization being thrown in your face has become so challenging! Although there is no clear-cut solution, as Crampton suggests (a standardized guideline may only add more problems), it is essential to account for both internal and now external ethics when using technology. Thus, technology can go beyond a "tool," as Sadhana pointed out, and expand to become a positive platform to uplift and voice underrepresented and vulnerable communities instead of targeting them.

(This is completely off-topic, but I just found the article about social media's downfall being from capitalism, and it being written by someone with the surname Marx, which was just so ironic.)

**Reply by stinest** [reply]
I completely agree, my initial understanding had me believing that technology and ethics were only loosely connected, but these readings were extremely eye-opening and made it easy to see the pervasive influence of capitalism and other social issues on technology. (I also found their last name so ironic and I'm so glad someone else pointed it out!!)

**Reply by sachigill** [reply]
I fully agree with you, I find it so hard now to explore the internet or any digital platform without some sort of profit driven strategy or ads directly in my face. I think those who work in these tech spaces, or those in power need to start viewing technology less as a neutral entity, but in combination with societal context and ideological values, so these tools can be used for good rather than harming our communities.

**Comment by kchhoeu** [comment]
I think for me prior to reading the articles I recognized that society and technology have a very skewed relationship. I thought technology is definitely needed to advance in society, including any type of technology both mechanical and physical, both it definitely needs to be monitored or just be used as something that can help but not be abused. But after reading the article I think that the lack of ethics,, especially in cartography is really interesting and kind of crazy. I think that being able to manipulate data and information of certain topics. I guess connecting it back to my life, as a artist and coming from a family of creatives the use of AI in design and creative jobs is such a bizarre thing to me, i feel that there is some benefit on the capitalistic side of things, less humans means less people to pay, but in my opinion, it doesn't have the same feel that a human work does. Capitalism in technology and creative fields do start to harm the communication and work of others.

**Reply by giaari15** [reply]
I agree, but I think it goes further than tech being monitored and needs to be deconstructed as its own separate structure. Also a side tangent on AI art, no matter the technical aspects of AI art, it cannot replicate individual human experience so it is hard to consider it art when it is probably at most on the same level as reference pictures. With the climate crisis as well, it is a greater ethical consideration of using AI at all if it consumes so much power.

**Reply by kimada62** [reply]
I agree with your thoughts on technology's place in society. We need to monitor how we introduce its use to society to ensure that we are keeping everyone's privacy and artistry safe. However, with new technology we have to accept that it is going to be abused at times. I think that we should still fight when these acts of technological abuse occur. If there is no pushback, technology will not longer do its job of enhancing our lives. In the case of Crampton's explanation of solutions and problems- while technology was once a solution to enhance our lives, it has the potential to become an even larger problem than what it was trying to solve.

**Reply by pwinnie2026** [reply]
Especially with AI art, I feel very similar, and I know a friend who personally wants to go into law school studying this issue. Since I am technically in an AI program at UCLA, we had an activity to explore different AI models and programs that made music. While they are not that great yet, I still personally disagree with the use of AI tools to create and sell art. Especially without any legislation or pushback, these programs will want to expand their capability to be used more industrially in the mediums that we normally consider as means of human expression. Seeing AI develop without these conversations is a little jarring in my perspective, and we can see this in other forms of media as well.

**Reply by jasminereyz** [reply]
I 100% agree with thinking that technology is something needed to advance as a society. I think especially with how dependent humans are on technology, it is hard to see us without it. Even myself, I do not know who I would be without Apple/Google Maps lol. Like you, I relate this to capitalism and how it can be abused, great insight!

**Comment by giaari15** [comment]
Technology is a tool in society so I approach my ethics as an add-on to the irl. I believe the sentiment that there is no ethical consumption under capitalism extends to the online and technology overall. As long as the goals of technology are profit-based or reactionary, the use that comes out of tech will more than likely be profit-based or reactionary. This is along the lines in the Crampton article where he discusses commodification where items, concepts, people, etc., are treated and presented as products for sale. Because of this, the transactional relationships between a person and their screen are exacerbated. However, the term "no ethical consumption" should not be a green light to consume as intended, but as an acknowledgement of the fundamental and systemic faults of capitalism itself. Beyond advocating for greater causes, on smaller-scale simply using tech as a tool for communication and community combats the capitalist mediums bit by bit. I believe there needs to be a greater fundamental upheaval to making ethical tech than guidelines that Crampton suggests where the focus is to be in the direct hands of people.

**Reply by nesslee123** [reply]
I really enjoyed your response! I completely agree with you in regards to the importance of making ethical tech because it shifts the responsibility from the people and places it on larger corporations and organizations. I also enjoyed how you mentioned the transactional relationship between people and technology in today’s society because it is such a prominent relationship witnessed which takes away from the enjoyment of social media and technology in general.

**Reply by caitlynn-hauw** [reply]
Periodt, Gia! I truly agree with your last sentence. It's impossible to mold our current capitalist system, with Crampton's guidelines, into a sustainable society; we are far beyond that point. When the foundation of our society is crumbling, the only solution is to dismantle the current one, and build a new one.

**Comment by Keiya-Wada** [comment]
I would say that my views ethics on society and technology before reading these articles were too optimistic and simple-minded. Although I knew about the "surveillance society" that Crampton mentions, I actually never once took it seriously. Every time I would go to a website, I would see a pop up asking me to allow cookies, and I would press "accept all cookies" simply because I didn't think about the consequences of my actions. In his article, he mentions that some companies have data about up to 95% of the U.S, which is absolutely mind-blowing. Even after reading these articles, my views seems to be conflicted, as it is ultimately up to how companies use those data. There is a possibility that they are working towards a powerful tool to empower communities, while the complete opposite may be the case. Crampton mentions how "the majority of GIS practice so far has operated from a positivist mentality" but seeing how society is forming to become more and more financially driven, I'm not sure if that will last. As such, I believe that the ethics that are explored in this class teaches us to be well informed and responsible as consumers of technological tools, and fight back against misuses of data and GIS.

**Reply by nesslee123** [reply]
Really great response! I completely relate to your experience with accessing websites and allowing for companies’ surveillance without considering the outcome. After reading the articles I understand how my data may be misused and misrepresented which is something I have not really considered, previously. It is also important to consider when the article was published since technology has significantly advanced in the last almost 30 years and when this article was published, companies had already accumulated data on 95% of the US.

**Reply by chrissanrow** [reply]
I think it is really concerning that the companies have such control over our data, and that we don't know if it is being used for causes that benefit or hurt us is even more concerning. I think that it is really important to be informed consumers of technology, as you mention, but I wonder if implementing more rigorous regulation on the usage of data will allow us to feel more confident in companies, the producers of technology, being informed and responsible.

**Reply by jasminereyz** [reply]
I also agree! I think there are a lot of things that we do not realize are happening especially within technology that is not like outright obvious? I definitely also always pushed "accept all cookies" without even know what the cookies were or what they meant. I also agree that I am glad I am taking this class because now I know how to recognize these issues

**Comment by nesslee123** [comment]
Technology is constantly advancing and being improved, but those improvements are not always beneficial to society and sometimes the harms outweigh the potential benefits. This is seen by the ways in which large corporations and organizations use social media to surveil the public which can lead to harmful misrepresentations of the people being surveilled. Jeremy W. Crampton’s (1995) article discussed this in detail. Crampton explains, “This can lead to inaccurate conclusions, a consequence of the ecological fallacy. For example, using a map of AIDS at the block level, insurance companies may charge higher premiums to people from blocks with high rates of AIDS. Note that individuals from one of these blocks are not the basis for this decision, but nevertheless, through their bad luck in being in that block, they are personally affected by aggregated data” (p. 87). This demonstrates the potential negative implications of technology in society, today. It is imperative that we begin to balance the scales of social justice and technological advancement because otherwise we continue to harm society. These ethical views empower civil society because they highlight the flaws and imbalances currently present and offer recommendations to drift the focus from technological advancement to improvements in social justice which will allow for a better civil society.

**Reply by sachigill** [reply]
I definitely agree with your response! One might think the advancement of technology directly correlates with the advancement and betterment of society, when in reality that is not true. As large corporations continue to use more of our data, this only further allows them to make harmful assumptions about populations, negatively impacting many communities. There needs to be a more cohesive understanding of societal context when handling our data, as well as the reduction of capitalistic tactics and only thinking about profit, otherwise things will only continue to get worse with the constant development of new technological tools.

**Reply by chrissanrow** [reply]
I agree with your sentiment that we need to balance the ethics and advancement of technology. I think it's interesting to think about your first statement, where you mention that technology is constantly advancing and being improved. As you mention, we need to ask if these improvements are really beneficial, and exactly who they are benefiting.

**Comment by sachigill** [comment]
Before these readings, I didn't realize how much the structures of capitalism were intertwined into our technology, and the overall commercialization of every aspect of the digital word. I think I have become numb to the abundance of ads and profit driven tactics I encounter every time I am on any sort of platform. I definitely feel these articles helped put things back into perspective for me. As technology continues to expand, I feel the gap between these developments, and considerations of ethics and societal context within these spaces, only further grow. I believe that when it comes to society and technology, our goals should be to enhance and uplift the communities around us, and create an open space of collaboration, not just fulfill capitalistic goals. As Crampton says, we as a society need to start considering not just the internal problems of GIS and data, but also focus on the external issues, and recognize the societal context and ideological framework behind everything, as it plays a huge role in ethical GIS practices.

**Reply by zhamic7** [reply]
I also agree that ads and consumer marketing tactics have become so commonplace on pretty much all major social media platforms that it's really easy to become numb to all of it. I suppose that is one challenge to ethics in technology, which is to recognize what specifically we've been convinced to accept and thus gotten used over time to should in actuality have been unacceptable in the first place.

**Comment by chrissanrow** [comment]
While it is often suggested and advertised that technology is meant to be neutral and objective, its creation, production, and implementation all involve and interact with civil society, necessitating an ethical discussion. As Crawford points out, the ethics of technology can often be broken down into "internal" and "external" analyses, which target different components of technology's life cycle. I believe that what should be prioritized is the external ethicality of technology, particularly in terms of how it affects individuals and communities. As both the readings reveal, a sort of techno-determinism causes us to hyperfocus on the internal, technical, and logistical ethics of technology. While these aspects are certainly important, they should be analyzed and adjusted with respect to the context that they exist in. The context of capitalism drives the surveillance, privacy, and power dynamic issues discussed in both articles - so as much as we may like to think that technology is an objective, sole solution for so many issues, we must first look at the reality of the social context it is manifested in.

**Reply by giaari15** [reply]
I agree social context drives how tech is utilized. I think surveillance is the most interesting one because there is a lot of rhetoric about regulation on data and such, but no real solution is possible as long as capitalism is the structure because money wins all. Corporations like Meta only need to make enough to pay fines.

**Reply by NellieNguyen149** [reply]
Nice response, I also included Crawford's outline of internal and external analyses when thinking of ethics, society, and technology. I enjoy the way you assign both internal and external perspectives to how they relate to multiple aspects of technology. I also appreciate how you mentioned the context of capitalism, which is a system that utilizes and prioritizes technology for increased efficiency and carrying out its purposes. By acknowledging the contexts and uses of technology in different lenses and not just assuming all use of technology is objective and the solution to issues, we get to understand the nuances behind its varying applications and how it is important to tie ethics into technological uses.

**Comment by jasminereyz** [comment]
I believe I have very similar views to other people my age, I feel that there are a lot of extremes when it comes to technology and the use of technology and it will always be abused. Before reading the article, I did not think of GIS as a tool of thinking and even before taking this class, I did not even know what GIS meant. 

I do believe that ethics are not always taken into consideration with technology, as we've learned that technology and mapping can be heavily biased and presented in ways for personal gains. It does make me wonder how we can even begin to tackle this problem when there is not much a physical problem and I think it is easy to fall hopeless with ethics and technology.

**Reply by NellieNguyen149** [reply]
You make a good point about viewing technology in different extremes. I personally also had an understanding of technology with an "extremity" mindset, where it was 100 percent right or 100 percent wrong. Usually, I noticed that I would have a rather close-minded perspective when it comes to using technology and that I took on a "it's a hopeless matter, so why even bother?" type of attitude that you allude to, so I'm doing my best to break free of that hopeless mindset and reconsider my initial perspective.

**Comment by NellieNguyen149** [comment]
Consider what are your ethical views when it comes to society and technology? How do your ethics empower civil society?

After reading the passages, I feel that my ethics when viewing society and technology is quite split: when approaching societal concepts and issues, I can better recognize both internal and external perspectives, as Crampton defines them. Because of this, I have a broader understanding of ethics in relation to society/the community, accounting for context, history, and the overall idealogical framework/implications. However, when approaching technology, I had a more 'black or white' perspective, thinking that the use of technology could either be 100 percent morally correct or 100 percent morally incorrect. 

Upon taking this class and finishing these readings, I understand that there is a bigger connection between society and the use of technology than I once thought, and that interpreting the use and intentions of technology isn't easily categorized into a "right or wrong" type of decision. I always assumed that people use and develop technology with their intentions set already before even starting: the "good" use of technology comes from people who set out to create something with good intentions, while the "bad" use of technology comes from people who set out to create something out of self-interest. However, I found that even with good intentions, technology isn't always produced to be ethical. This idea is briefly discussed by Crampton with "Geographic information systems are ethically complex, and often demonstrate 'ethically inconsistent' behavior (Curry 1995) due to overlapping and competitive goals." I realize it is nearly impossible to create a 100 percent ethical use of technology, so instead of trying to chase after that goal, it would maybe be more beneficial for me to take a step back and re-evaluate my understanding of ethics in technology/GIS. I will try to view ethics in technology the way I understand ethics in society: a complex discipline that requires different viewpoints and a multifaceted approach.

**Reply by caitlynn-hauw** [reply]
I think your response is really thoughtful. While I take the time to re-evaluate my understanding of ethics in technology and GIS—as you similarly are—for me, it's important to consider how you're able to empower others in a fundamentally unethical system. If capitalism creates a system where there is no ethical consumption under it, as Gia said, then how can we fight against that? I think the work we are doing in this class is helping us answer that question.

**Comment by caitlynn-hauw** [comment]
The reason I have had no intention of pursuing a full-time career in technology is because I believe it is so easy to become absorbed by how lucrative the technology industry can be, and therefore disregarding your ethics in the process; this isn't a testament to our own failings or the selfishness of the entirety of the human race, but a sinking ship that's becoming too full of water. As aptly-named Marx in our reading said, this is a symptom—not of bad technology, as many would like us to believe—but of our capitalist society. Contrary to what techno-detminists want us to believe, technology merely responds to "the economic incentives of our capitalist system." If we believe in bad technology, it's emblematic of a society that has failed us. 

I think the "most ethical technology" comes from what is community-owned and open source. My own ethics reflect this observation. In the short term, in order to empower a community, you must give them ownership over the solutions to their own problems. However, in our current system, I believe long-term empowerment only comes from dismantling our current system that was built to fail, and creating a new one.
